text/0000-image-retention-policy.md

```diff
@@ -0,0 +1,56 @@
+# Define an Image Retention Policy for Paketo Images
+
+## Summary
+
+At the moment, the Paketo Project is maintaining images (buildpack, builder, stack, etc..) since the beginning of the project. This RFC proposes that we define an image retention policy that would allow the project to delete old images.
+
+## Motivation
+
+Multiple reasons:
+
+1. It is impractical from a cost and resources perspective for the project to maintain all images throughout the history of time. Time continues on infinitely, and eventually enough resources will accumulate to where this becomes a burden to the project.
```

Comment:
Given this is equally true of dependencies built and provided by the Paketo project I think we should include deps in any retention policy we might adopt (cc @paketo-buildpacks/dependencies-maintainers)